lib/main.js

```diff
@@ -64,7 +70,7 @@ const render = App => ReactDOM.render(
   document.getElementById('main')
 )
 
-render(TrimetWebapp)
+hot(render(TrimetWebapp))
```

Comment:
This probably deserves a comment. It would appear that hot loading would be enabled at all times based on how this is called.

Reply:
Ah, I see now that this is to keep the state in place during a hot reload. Still deserves a comment and description of dev vs. prod.